_posts/2021-05-04-how-to-update-your-ref-arch.adoc

```diff
@@ -27,13 +27,18 @@ endif::[]
 == Intro
 
 This guide walks you through how to update v1.0 of the https://gruntwork.io/reference-architecture/[Gruntwork Reference
-Architecture] to use the
+Architecture], which uses a private `infrastructure-modules` repo, to v2.0, which uses the
 https://blog.gruntwork.io/introducing-the-gruntwork-module-service-and-architecture-catalogs-eb3a21b99f70[Gruntwork
 Service Catalog]. The Gruntwork Service Catalog is a set of dozens of highly configurable, battle-tested,
 production-grade _services_ that you can deploy off-the-shelf without writing any code. Many modules offered in the
 `infrastructure-modules` repository of the Gruntwork Reference Architecture are fully replaced with the Service Catalog,
 eliminating the need to maintain snapshots of the Services in your own repository.
 
+[.exceptional]
+TIP: The Landing Zone migration guide will be published soon.
```

Comment:
I'm personally not keen on writing a Landing Zone migration guide. Customers that want to adopt the Landing Zone can follow [this guide](https://gruntwork.io/guides/foundations/how-to-configure-production-grade-aws-account-structure/). But I think others might feel differently from me on this matter.